docs/tests/integration/test_wazuh_db/index.md

```diff
@@ -1,3 +1,13 @@
 # Overview
 
+**Wazuh-db** is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket so receive requests and provide information.
 
+## Tiers
+### Tier 0
+#### Test wazuh_db
+
+These tests check the Wazuh-db commands syntax, valid responses and error messages in different circumstances. They are divided in **.yaml** configuration files:
+
+- **agent_messages.yaml**: tests for agents in general, currently only for Agents' CVEs table
+- **fim_messages.yaml**: tests for agents, but only for FIM module
```

Comment:
Maybe we can express this like:
_tests for agents DB, FIM module commands._

Reply:
OK, fixed!